gwentifyHandler.py

```diff
@@ -92,14 +102,16 @@ def getCardJson(html):
 
         # We check the name and do the appropriate action to store it in the map.
         if attribute == "Group:":
-            dataMap["type:"] = data.a.get_text().replace(u'\u00a0', u' ').strip()
+            dataMap["type"] = data.a.get_text().replace(u'\u00a0', u' ').strip()
```

Comment:
I think these were typos? I assume you don't want the ":" to be part of the key.